shared/js/background/atb.es6.js

```diff
@@ -176,47 +173,18 @@ const ATB = (() => {
             // - the user wasn't already looking at the app install page
             // - the user hasn't seen the page before
             settings.ready().then(() => {
-                // Special case for the cross product promotion on desktop (cppd) experiment
-                if (settings.getSetting('isMultiStepOnboarding')) {
```

Comment:
wait, this experiment is gone? is this a unrelated cleanup?

Reply:
Yes, this is an unrelated cleanup. All the cleanup is done in [this commit](https://github.com/duckduckgo/duckduckgo-privacy-extension/pull/568/commits/c42419d88fc0f1aacf49f6921f857ac624a02e7d).